tools/add_fib.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+
+# CMSIS-DAP Interface Firmware
```

Comment:
cmsis-dap ? in tools? What does script do? There's no documentation for this script (I cant locate any comments in this file (for API for instance). Lot of commented dead code, should not be there. Values like vectors are predefined, I assume this is just for one target, or is it generic? based on the location and t he name I would say it could be used by any other target?